mantidproject/mantid
# Add adapter for ISIS Command Layer

**Description of work.**

*Builds on #27680*

Adds:
- additional missing tags found in the command interface and tests
- a new adapter class 
- inserts it into the location where we could do translation (so I don't forget)

**To test:**
- Ensure the new unit tests run
- Code review


Fixes #27681 .

*This does not require release notes* because this is part of epic #18826

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
